charts/dsv-syncer/Chart.yaml

```diff
@@ -9,7 +9,7 @@ keywords:
   - secrets
   - vault
 type: application
-version: 0.1.0
+version: 0.1.1
 appVersion: latest
 maintainers:
   - name: Adam Migus
```

Comment:
@sheldonhull should this be updated to you?

Reply:
@tylerezimmerman sorry, what do you mean? I bumped the chart version to reflect minor changes. 

If you want to suggest an edit or removal of something, use the "inline suggestion" button on the comment box, tweak it and submit and it will let me accept/reject the edit.